docs/jazzy.yml

```
@@ -112,3 +125,30 @@ custom_categories:
       - RouteSnappingMinimumHorizontalAccuracy
       - RouteSnappingMinimumSpeed
       - NavigationSettings
+  - name: User Feedback collection
```

This is a parent item, so it’s inherently a collection of children items:

```suggestion
  - name: User Feedback
```

---

Originally I meant 'collection' as 'a process of collecting', but turns out that is not obvious :)